web/src/features/fba/components/fbaMap.test.tsx

```diff
@@ -1,14 +1,15 @@
 import { render } from '@testing-library/react'
 import store from 'app/store'
 import FBAMap from 'features/fba/components/FBAMap'
+import { DateTime } from 'luxon'
 import React from 'react'
 import { Provider } from 'react-redux'
 
 describe('FBAMap', () => {
   it('should render height with height and width properties set', () => {
     const { getByTestId } = render(
       <Provider store={store}>
-        <FBAMap selectedFireCenter={undefined} className={''} />
+        <FBAMap date={DateTime.fromISO('2016-05-25T09:08:34.123')} selectedFireCenter={undefined} className={''} />
```

Comment:
Can we use just the date part, instead if a time?
```suggestion
        <FBAMap date={DateTime.fromISO('2016-05-25')} selectedFireCenter={undefined} className={''} />
```